components/input-number/index.tsx

```diff
@@ -74,6 +74,7 @@ export default class InputNumber extends React.Component<InputNumberProps, any>
           return (
             <RcInputNumber
               ref={this.saveInputNumber}
+              type="tel"
```

Comment:
Why set `tel` here?

Reply:
https://github.com/ant-design/ant-design/issues/21199#issuecomment-582269009

Reply:
Hmm...this may affect accessibility.

Reply:
Some how?

Reply:
Will tread input as a tel input.